draft-ecahc-moderation.md

```diff
@@ -121,6 +121,14 @@ has, mostly for historic reasons, a team of moderators that are not
 list administrators and operate by a different set of processes
 {{?RFC9245}}.
 
+Note that the term "moderation" can refer both to *preemptive*
+moderation, where a moderator reviews messages before release
+to the mailing list, and *reactive* moderation, where the moderator
+intervenes after a message has been sent. The IETF mainly practices
+reactive moderation, with a spectrum from gentle reminders on- and
+off-list, all the way to suspension of posting rights and other ways
+of participating or communicating.
```

Comment:
This sounds like we want to retain sticking to mostly reactive moderation? I think we want to leave this up to the moderation team to deicide as part of their process definition.